### PR TITLE
978, Part 1, Creating onboarding modal component

### PR DESCRIPTION
**Pull request by judeamos**
This is part 1 of my PR. These changes will allow other team members to start working on their stories, as they were dependent on mine.

Changes: 
1. I've created a folder called UserOnboardingModal and corresponding files, that will house all onboarding processes/pages situated in the modal.
2. I've modified me.js where the UserProfilePage component renders the UserProfile component, to show the modal when the username of the user is empty. (This will be refined in later PRs).
3. I have created the basic styling of the UserOnboardingModal in the StyledUserOnboardingModal.js file. 

I will later create another PR for part 2 of this story.

### CODE REVIEW

File: apps/app/pages/users/me.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Page from '@devlaunchers/website/src/pages/users/me';
 import IdeaApp from '@devlaunchers/website/src/pages/_app';
-export { getStaticProps } from '@devlaunchers/website/src/pages/index';
+// export { getStaticProps, getStaticPaths } from '@devlaunchers/website/src/pages/users/me';
```
Comment (judeamos) [inline apps/app/pages/users/me.js:3]: Test comment, remove comment

**Review by judeamos — COMMENTED** [review]
I've had a look at your changes (test comment for tutorial)

File: apps/website/src/components/modules/UserOnboardingModal/UserOnboardingModal.js [hunk 2]
```diff
@@ -11,8 +11,8 @@ Modal.setAppElement("#__next");
  * @description This is custom modal for the user onboarding. 
  */
 export default function UserOnboardingModal({isOpen}) {
-    const [modalIsOpen, setModalIsOpen] = useState(isOpen);
-    
+    const [modalIsOpen, setModalIsOpen] = useState(true);
```
Comment (erinsgithubpage) [inline apps/website/src/components/modules/UserOnboardingModal/UserOnboardingModal.js:14]: Looks good, Jude!

**Review by erinsgithubpage — COMMENTED** [review]
This code looks good, Jude. From what I see, it all equates greatly.

**Review by HavreLoic — COMMENTED** [review]
Great, everything looks like it is working

**Pull request merged**